### ISSUE TITLE
Rooms and ongoing battles are too similar on the list of battles

### ISSUE THREAD

**Issue by Sgathec** [ZeroK-RTS/Chobby#211]
Please differentiate the custom room and ongoing battle "buttons" in a better fashion. I think clicking on any such "button" that doesn't represent a custom room causes Zero-K to start loading. Once started, the loading cannot be canceled, and it's probably pretty easy to commence accidentally.

**Comment by Licho1** [comment]
confirm window?

**Comment by Sgathec** [comment]
Yes, but it should probably have a "remember my choice" option in my opinion. However, I don't it's a very good way to remedy the situation.

**Comment by Licho1** [comment]
Also the rooms are all actually joinable.. but those that currently auto start game are rooms which cannot start again (they will close after game ends).

**Comment by GoogleFrog** [comment]
The matchmaker rooms should never reveal that they can technically be joined and manipulated like a custom room. One issue with quickmatch in the past is that people would join the room expecting to be able to play games like they would a custom game.
